### PR TITLE
Added TestApp changes for WPF app

### PR DESCRIPTION
**Pull request by chandan-anjani**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the Windows code?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

A few sentences describing the overall goals of the pull request.

## Related PRs or issues

List related PRs and other issues.

## Misc

Add what's missing, notes on what you tested, additional thoughts or questions.

### CODE REVIEW

File: Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs [hunk 1]
```diff
@@ -56,6 +57,12 @@ public MainWindow()
             {
                 CountryCodeEnableCheckbox.IsChecked = true;
             }
+
+            if(Settings.Default.UserId is string id)
```
Comment (guperrot) [inline Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs]: reformat file in VS

Comment (chandan-anjani) [reply Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs]: Done

File: Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs [hunk 2]
```diff
@@ -293,5 +298,26 @@ void HandleOrThrow(Action action)
...
+
+        private void UserId_KeyDown(object sender, System.Windows.Input.KeyEventArgs e)
+        {
+            if (e.Key == Key.Return)
+            {
+                var userId = UserId.Text;
+                var text = string.IsNullOrEmpty(userId) ? null : userId;
+                AppCenter.SetUserId(text);
+                Settings.Default.UserId = text;
+                Settings.Default.Save();
```
Comment (guperrot) [inline Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs]: Those lines are the exact same as UserId_lostFocus, should use a private method for both

Comment (chandan-anjani) [reply Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs]: Make sense. Added a private method.

**Pull request merged**